advocacy_docs/supported-open-source/patroni/01-supported_topology.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Supported Topology'
+description: 'EDB supported topology'
+tags:
+  - topology
+  - Patroni
+---
+
+### Description
+
+Main HA components:
+* PostgreSQL cluster: the database cluster, usually consisting of a primary and two or more replicas
+* Patroni: used as the failover management utility
+* etcd: used as a distributed configuration store (DCS), containing cluster information such as configuration, health, and current status
+
+![Supported topology](images/supported-arch.png)
+
+#### PostgreSQL
+
+EDB currently only supports using Patroni and PostgreSQL _Streaming Replication_ (physical) with a primary server and two or more replicas.
+
+#### Patroni
+
+Patroni is installed on the PostgreSQL nodes. EDB supports the [latest release](https://patroni.readthedocs.io/en/latest/releases.html#releases).
+
+#### Etcd
+
+Etcd is a strongly consistent, distributed key-value store that provides a reliable way to store data that needs to be accessed by a distributed system or cluster of machines.
+
+Head over to [https://etcd.io](https://etcd.io/docs/) documentation for more information.
+
+!!! Note
+  For production environments, it is recommended to setup a three nodes cluster. One etcd cluster can serve multiple PostgreSQL/Patroni clusters.
```

Comment:
Two observations:
. "a three-node cluster" or "a cluster of three nodes"
. We should mention that the etcd nodes need to be independent from the Postgres nodes

Reply:
adding _Etcd nodes should be installed independently from the Postgres nodes (not on the same hosts)._

Reply:
fixed in 5bf5f28